content/posts/a-guide-on-commit-messages.mdx

```diff
@@ -110,10 +110,45 @@ Co-authored-by: another-name <another-name@example.com>"
 
 ## Commit conventions
 
-There are a few conventions that comes with tools to facilitate your daily life. CLIs to create commits message and changelogs. There are well documented so you don't have to write documentation about your own commit message convention.
+As we talked in the previous section you should define a commit convention for your team. It exists open source commit convention that could be a good source of inspiration. They also come with a whole ecosystem that includes tools to help you write commit messages, generate changelog, create releases, ... A lot of things that can take a large amount of time. There are well documented so you don't have to write documentation about your own commit message convention.
 
-### conventionnal commits
-- Monorepo
+We will talk about two of them [Conventional Commits](https://conventionalcommits.org) and [gitmoji](https://gitmoji.carloscuesta.me/).
+
+### Conventional Commits
+
+It is a specification inspired from [Angular commit message guidelines](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines). It features a few interesting rules like:
+
+- Commit must be prefixed with a type (feat, fix)
+- A scope may be provided to point out a specific section a the codebase (really interesting for monorepos)
```

Comment:
a specific section a the codebase 
Little typo here

Reply:
Still present.

Reply:
I corrected it in my last commit